build-support/bin/go/mac/10.10/1.12.7/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Each of the OSX version directories can probably be symlinks to just one copy of this script for this version: see c803d9f4b3dc9d1bc8e9d662a7795a2c511f8a99 for an example.

Reply:
Thanks - changed that

Reply:
there we go!